webpack.config.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
+
+var packageData = require('./package.json');
+
+var filename = [packageData.name, packageData.version, 'js'];
+
```

Comment:
I was a little confused by the instructions since it didn't seem like the webpack variable was used for anything. Maybe in a later lesson it is used? Everything seems to work so far.